prboom2/src/SDL/i_system.c

```diff
@@ -254,11 +254,27 @@ const char *I_DoomExeDir(void)
 
 const char* I_GetTempDir(void)
 {
-  static char tmp_path[PATH_MAX] = {0};
+  static const char* tmp_path = NULL;
```

Comment:
`static` values are always zero-initialized, so you can leave off the assignment.